ui/types/ember-simple-auth/services/session.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module 'ember-simple-auth/services/session' {
+  interface SessionService {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    authenticate(authenticator: string, params: any): Promise<void>;
+    isAuthenticated: boolean;
+    invalidate(): Promise<void>;
+    data: SessionData;
+  }
+
+  interface SessionData {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    authenticated?: any;
```

Comment:
Could we use `unknown` rather than `any` in the spots? Or does that just add clutter?

Reply:
I think that would work. I'm still sometimes confused by the difference between those two in ts.

Reply:
Yeah me too. I think it’s that `unknown` forces you to check what type a thing is before you use it (or at least check that it conforms to some interface).